src/components/Card.js

```diff
@@ -0,0 +1,23 @@
+import './Card.css'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTrashCan} from '@fortawesome/free-solid-svg-icons'
+
+function Card(props){
+    return(
+        <div className="card" onDoubleClick={()=>{props.openModalUpdate(props.data.id)}}>
+            <div className='row align-center justify-between default-color border-box width-100  card-content'>
+            <div className='row align-center card-title'>
+            <input type="checkbox" className="checkbox"/>
+            <h3>{props.data.title} : {props.data.priority.toUpperCase()}</h3>
```

Comment:
Use lodash get to extract properties nested 2 levels deep and have fallbacks for nullish cases